extensions/dicom-pdf/src/getSopClassHandlerModule.js

```diff
@@ -0,0 +1,66 @@
+import { SOPClassHandlerName, SOPClassHandlerId } from './id';
+import { utils, classes } from '@ohif/core';
+
+const { ImageSet } = classes;
+
+const SOP_CLASS_UIDS = {
+  ENCAPSULATED_PDF: '1.2.840.10008.5.1.4.1.1.104.1',
+};
+
+const sopClassUids = Object.values(SOP_CLASS_UIDS);
+
+
+const _getDisplaySetsFromSeries = (instances, servicesManager, extensionManager) => {
+  console.log('Called getDisplaySetsFromSeries', instances);
+  const dataSource = extensionManager.getActiveDataSource()[0];
+  return instances
+    .map(instance => {
+      const { Modality, SOPInstanceUID, EncapsulatedDocument } = instance;
+      const { SeriesDescription, MIMETypeOfEncapsulatedDocument, } = instance;
+      const { SeriesNumber, SeriesDate, SeriesInstanceUID, StudyInstanceUID, NumberOfFrames } = instance;
+      console.log('Display PDF of type', MIMETypeOfEncapsulatedDocument, EncapsulatedDocument);
+      const pdfUrl = dataSource.retrieve.directURL({ instance, tag: 'EncapsulatedDocument', defaultType: MIMETypeOfEncapsulatedDocument });
```

Comment:
do we have directURL implementation for other dataSources? or we just throw error here if the dataSource is not dicomWebDataSource?

Reply:
Working on updating this one - needs a change to dicomweb-client